app/assets/translations/appConstants/bebbo/RSen.ts

```diff
@@ -24,29 +24,29 @@ const RSen = {
                 "tNcheader": "Terms and Conditions",
                 "tncScreenContent": "Bebbo does not collect or store any personal information. All the information you enter is saved only on your phone.",
                 "tNccheckbox1": "I have read, understood and accepted the Terms and Conditions of the Bebbo application",
-                "tncCheckBoxText": "Terms and Conditions of the Bebbo application",
-                "tNccheckbox2": "I have read, understood and accepted the ",
+                "tncCheckBoxText": "Terms and Conditions of the Bebbo application ",
+                "tNccheckbox2": "I have read, understood and accepted the",
                 "tNcprivacyPolicy": "Privacy Policy of the Bebbo application",
                 "tNccheckbox3": "I understand that my usage of the app may generate anonymized data, which may be used to improve Bebbo. While providing this data is not mandatory, I recognize that it can help enhance the overall user experience.",
                 "tNcprivacyPolicyTitle": "Privacy Policy",
                 "tNcacceptbtn": "I accept the Terms and Conditions",
                 "loadingText": "Fetching content…",
-                "childSetupheader": "Please take a moment to personalize your app",
-                "childSetupdobLabel": "Child's date of birth",
+                "childSetupheader": "Get the most out of Bebbo!",
+                "childSetupdobLabel": "When is the child’s birth date or due date?",
                 "childSetupdobSelector": "Enter date of birth / expected due date",
-                "childSetupdueLabel": "Original due date",
+                "childSetupdueLabel": "What was the child’s expected birth date?",
                 "childSetupdueSelector": "Enter original due date",
-                "childSetupprematureLabel": "Baby born prematurely",
+                "childSetupprematureLabel": "The child was born prematurely.",
                 "childSetupprematureMessage": "A baby born before 37 weeks of pregnancy is considered premature or born too early.",
-                "childSetuprelationSelectTitle": "Your relationship to the child",
+                "childSetuprelationSelectTitle": "You are:",
                 "childSetuprelationSelectText": "Select",
                 "childSetupcontinueBtnText": "Continue",
-                "childSetupListheader": "Your child's profile has been created!",
-                "childSetupListsubHeader": "Please review and add siblings if you have other children between 0-6 years old. ",
-                "childSetupListaddSiblingBtn": "Add sister or brother",
+                "childSetupListheader": "Bebbo is getting ready to support your parenting journey!",
+                "childSetupListsubHeader": "If you have any other children aged 0 to 6, you can add them now or later.",
```

Comment:
You can add them now or later, if you have any other children aged 0 to 6.
looks more correct but I think you are just using the string files so it is fine to keep as it is.